src/vector/crypto.rs

```diff
@@ -132,7 +132,7 @@ fn generate_normalized_vector(
     calculate_normalized_vector(multivariate_normal_sample, uniform_point_in_ball)
 }
 
-pub(crate) fn encrypt<R: RngCore + CryptoRng>(
+pub(crate) fn encrypt<R: RngCore + CryptoRng + Send + Sync>(
```

Comment:
What were the Send + Sync additions for?

Reply:
Cargo clippy started complaining about it.